Dev/VideoChatClient/VideoChatClientException/VideoChatClientStatus.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <ostream>
+
+enum class VideoChatClientStatus
+{
+	AUTO_CLOSE_SOCKET_SOCKET_FAILED,
+	AUTO_INIT_WINSOCK_WSA_STARTUP_FAILED,
+};
+
+const char* to_string(const VideoChatClientStatus& e)
```

Comment:
Do we really want this function? for each status to return the string it represents? if we have hundreds of statuses it would be tough. You can binary search at the Status header file to see the given status. I think there are automatic things that may do this.